fades/main.py

```diff
@@ -154,6 +154,18 @@ def detect_inside_virtualenv(prefix, real_prefix, base_prefix):
     return prefix != base_prefix
 
 
+def _get_normalized_args(parser):
+    # support the case when executed from a shell-bang, where all the
+    # parameters come in sys.argv[1] in a single string separated
+    # by spaces (in this case, the third parameter is what is being
+    # executed)
+    env = os.environ
+    if ('_' in env and env['_'] != sys.argv[0] and len(sys.argv) >= 1 and " " in sys.argv[1]):
```

Comment:
please remove unnecessary `()` around the comparisons

Reply:
Done